app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,88 +4,112 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
     color : $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
```

Comment:
No it is the opposite: we don't want the user to see the label, but we want screen readers to read them, that's why we use this instead of a simple display: block;